app/assets/stylesheets/application.scss

```diff
@@ -59,6 +59,9 @@ strong {
       content: None;
       display: block
     }
+    .govuk-summary-list--half .govuk-summary-list__key {
+       width: 50%;
+    }
```

Comment:
This matches the prototype but didn't load in on local without adding it to our project's file